managed/ui/src/components/tables/CreateBackup/CreateBackup.js

```diff
@@ -519,14 +519,12 @@ export default class CreateBackup extends Component {
                       type="number"
                       label={'Parallel Threads'}
                     />
-                    {isScheduled && (
-                      <Field
-                        name="timeBeforeDelete"
-                        type={'number'}
-                        component={YBFormInput}
-                        label={'Number of Days to Retain Backup'}
-                      />
-                    )}
+                    <Field
```

Comment:
@yb-andrew pls check this change as you most likely more familiar with backups code